processor/src/test/java/org/mapstruct/ap/test/annotatewith/deprecated/newer/DeprecatedTest.java

```diff
@@ -0,0 +1,123 @@
+/*
+ * Copyright MapStruct Authors.
+ *
+ * Licensed under the Apache License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package org.mapstruct.ap.test.annotatewith.deprecated.newer;
+
+import java.lang.reflect.Method;
+
+import org.junit.jupiter.api.condition.EnabledForJreRange;
+import org.junit.jupiter.api.condition.JRE;
+import org.mapstruct.ap.test.annotatewith.CustomMethodOnlyAnnotation;
+import org.mapstruct.ap.test.annotatewith.deprecated.older.OlderDeprecatedMapperWithClass;
+import org.mapstruct.ap.test.annotatewith.deprecated.older.OlderDeprecatedMapperWithMethod;
+import org.mapstruct.ap.test.annotatewith.deprecated.older.OlderRepeatDeprecatedMapper;
+import org.mapstruct.ap.testutil.ProcessorTest;
+import org.mapstruct.ap.testutil.WithClasses;
+import org.mapstruct.ap.testutil.compilation.annotation.CompilationResult;
+import org.mapstruct.ap.testutil.compilation.annotation.Diagnostic;
+import org.mapstruct.ap.testutil.compilation.annotation.ExpectedCompilationOutcome;
+import org.mapstruct.factory.Mappers;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+/**
+ * @author orange add
+ */
+public class DeprecatedTest {
```

Comment:
I see that there is some small misunderstanding of the use of the integration `fullFeatureTest`.

The full feature integration test does nothing more than compiling all the mappers that are in our processor test module. However, it doesn't run any tests on its own. 

In addition to that, our processor tests are currently only running with Java 11 and later. Which means that we can easily move this test to the package above `org.mapstruct.ap.test.annotatewith.deprecated` and remove the one from `org.mapstruct.ap.test.annotatewith.deprecated.older`. I would also remove the `@EnabledForJreRange` from this tests, since it is not needed.

Can we also please change the names of the mappers a bit. e.g. everything from the `older` package can just be moved to the package above `org.mapstruct.ap.test.annotatewith.deprecated` and the prefix `Older` can be removed. Everything in the `newer` package can be moved to a different package called `java11`, remove the prefix `Newer` and do things like `DeprecatedWithAttributesMapper`, etc. 

I am asking for this in order to reduce some of the additional test complexity and to be immediately clear when someone reads the names of the classes.

Reply:
Thank you for your patient guidance, I will correct it